aanouncing-python-sdk-ga.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Announcing General Availability for Xata's Python SDK'
+description: 'We are excited to ship the first stable version of our Python SDK'
```

Comment:
Maybe instead of "stable" we say GA. Stable indirectly implies that all other versions were not stable (which may in fact be the case, but we don't need to let everyone know that 😉)

Reply:
Sounds good! Adjusted